com.logseq.Logseq.yml

```diff
@@ -54,7 +115,17 @@ modules:
       - type: script
         dest-filename: run.sh
         commands:
-          - zypak-wrapper.sh /app/logseq/Logseq "$@"
+          - |
+            #!/bin/env bash
+            set -e -u -o pipefail
+            FLAG_FILE=".logseq/update_warning_shown"
```

Comment:
Does relative path work? Maybe we can use `$XDG_CACHE_HOME/flatpak_update_warnings/0.9.4-1` so that

1. The files are created in known location that does not conflict with upstream files
1. We can reuse the mechanism for other warnings in future